app/views/contexts/summary/question.py

```diff
@@ -55,13 +65,14 @@ def get_answer(self, answer_store, answer_id):
     def _build_answers(
         self,
         *,
-        answer_store,
-        question_schema,
-        block_id,
-        list_name,
-        return_to,
-        return_to_block_id,
-    ):
+        answer_store: AnswerStore,
+        question_schema: QuestionSchemaType,
+        block_id: str,
+        list_name: Optional[str],
+        return_to: Optional[str],
+        return_to_block_id: Optional[str],
+    ) -> Union[list[Mapping[str, Any]], list[dict[str, str]]]:
```

Comment:
Is this union needed?. I can just be list of dicts?

Reply:
👍